CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - move the `package:pedantic` to dev depencies #94
 - Added GH Action Changelog verifier #95
 - new Dart code file structure #96 
+- Base the sdk name on the platform (`sentry.dart` for io & flutter, `sentry.dart.browser` in a browser context) #103 
```

Comment:
As a note, we want to pass `sentry.dart.flutter` on the `sentry_flutter` package. We need to pass that as an argument when initializing the SDK through Flutter. That will land when we get a `SentryFlutter.init` of sorts

Reply:
yes I saw this in the issue, I noted it in the "Next steps". For now it will just be sentry.dart on mobile and sentry.dart.browser with Flutter web.

Reply:
Ohh that's a good point! We wouldn't want `.browser` on Dart Web :)
I'll add a note to the issue